global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.200"
+    "version": "2.1.201"
```

Comment:
Netcore 2.1 has just been released (sdk 2.1.300, MSFT versioning rules). Should we use that?

Reply:
I know, but this just aligns with FCS 23.0.1.
Yes, we should use it, but the change to dotnet core 2.1 is bigger and it should be in it's own PR, as the API surface may have changed.

Reply:
I see you already made that PR to switch to 2.1, so I've updated FCS-Fable too.